geo-types/src/line_string.rs

```diff
@@ -191,6 +191,11 @@ impl<'a, T: CoordNum> DoubleEndedIterator for CoordinatesIter<'a, T> {
 }
 
 impl<T: CoordNum> LineString<T> {
+    /// Instantiate Self from the raw content value
+    pub fn from_raw(value: Vec<Coordinate<T>>) -> Self {
```

Comment:
I've typically seen `from_raw` used as a name in Rust when using raw pointers. What do you think about just calling the constructor `new`?

Reply:
https://doc.rust-lang.org/nightly/std/?search=from_raw

Reply:
@frewsxcv yeah, i'm not too happy about the name either.  The problem with `new` is that we should have a consistently named static function on all geo types for instantiating, and `new` is already taken at least for some of them.  An alternative of course is to switch to a macro

Reply:
Can you clarify what you mean by "`new` is already taken at least for some of them"?

Reply:
@frewsxcv 3 existing structs already have the new method. For some reason I thought more geotypes had it, so it seems it might be ok to add more.
* `Point::new(x: T, y: T)`
* `Polygon::new(mut exterior: LineString<T>, mut interiors: Vec<LineString<T>>)`
* `GeometryCollection::new()`